.github/dependabot.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+updates:
+  - commit-message:
+      prefix: chore
+    directory: /backend
+    package-ecosystem: gomod
+    schedule:
+      interval: monthly
+    groups:
+      backend-golang-dependencies:
+        patterns:
+          - "*"
+    reviewers:
+      - "mendersoftware/backend-dependabot-reviewers"
+  - commit-message:
+      prefix: chore
+    directory: /backend
+    package-ecosystem: docker
```

Comment:
we can try, but I'm afraid this won't pick up the versions in the per service Dockerfiles... looking at https://github.com/dependabot/dependabot-core/issues/2178 we will likely have to list the directories here.

Reply:
yes, I'm preparing config with all `directories` listed